lol-frontend/src/config/store/player/reducer.ts

```diff
@@ -0,0 +1,21 @@
+import { editPlayerPosition } from "./actions";
+import { PlayerState, PlayerActionTypes } from "./types";
+
+const initialState = {
+  position: [0, 0]
+};
+
+const playerReducer = (
+  state = initialState,
+  action: PlayerActionTypes
+): PlayerState => {
+  const { position } = state;
+  switch (action.type) {
+    case "EDIT_PLAYER_POSITION":
+      return editPlayerPosition(position);
+    default:
+      return state;
+  }
+};
+
+export default playerReducer;
```

Comment:
In the reducer we want to make sure we are bringing in any state models (or types if that's what your familiar with) as well as the exported methods from our action model (or type).

Then we want to make sure the initialState is set to the type of whatever model we are using. 

So if you defines a Player model and it looked like so:
interface PlayerState {
   position: number[]
}

Then you would initialize your initialState like so:
const initialState: PlayerState = {
   position: [0,0]
}

** Tip: When you use a model you have to define all the items in the model unless you notate them as optional like:
interface Player {
    position?: [0,0]
}

you initialized the reducer correctly.

Whenever you return in the switch you should just be able to return the action.payload or spread the state and address one value at a time.